### ISSUE TITLE
Dynamic (slowing down) eta or learning rate

### ISSUE THREAD

**Issue by run2** [dmlc/xgboost#1283]
Hi
Is it possible to have a dynamic eta such that we can optimize on the minima. Ideally it should adjust with the gradient - like slow down as we reach close to the minima.

**Comment by pommedeterresautee** [comment]
it has been  in some way implemented.

**Comment by run2** [comment]
Can you provide some details on how we can use this feature and how it was implemented also which version ?

**Comment by pommedeterresautee** [comment]
Last comment https://github.com/dmlc/xgboost/issues/1310
